### ISSUE TITLE
Better text/plain version of the summary email

### ISSUE THREAD

**Issue by bzg** [entrepreneur-interet-general/bulletins#109]
**Problem: the `text/plain` version could be more readable**

When emails are both in `text/plain` and `text/html`, my email client displays the `text/plain` version by default. The `text/plain` version of `ma-semaine` emails is not as readable as it could be.

**Don't use markdown and utf-8 emojis**

1. Use plain text formatting instead of markdown for `text/plain`: e.g. replace `**` by `*`, etc. 
2. Use a old school plain/text smileys instead of utf-8 emojis.

**Other people might need this**

Since this application is likely to be used by developers and because some of them are likely to display `text/plain` versions of emails, this could be a real plus.

**Comment by AntoineAugusti** [comment]
Thanks for the report. It's not a major issue because it doesn't concern a majority of users and can be resolved through configuration. The layout doesn't seem too much broken for now in plain text.

Can you share a screenshot to see what it looks like?

**Comment by bzg** [comment]
Thanks for the feedback. Here you go:

![2019-08-19-104533_1920x1080_scrot](https://user-images.githubusercontent.com/63500/63251696-8714f700-c26e-11e9-9298-d9544fc8a403.png)

The question is: why using `markdown` in plain text? Is it something you need to let users add some formatting?

**Comment by AntoineAugusti** [comment]
The [email view](https://github.com/entrepreneur-interet-general/bulletins/blob/master/resources/views/emails/report.blade.php) is written in Markdown and rendered in HTML by Laravel. It seems Laravel sends the unrendered version of the Markdown source as the plain text version fo the email.

**Comment by bzg** [comment]
Ah, ok. I guess nobody cares about plain text enough to tell Laravel to DTRT.

**Comment by AntoineAugusti** [comment]
I'd say is not too bad as it is! Doing automatically another thing would require a decent amount of code and would be subjective.

**Comment by bzg** [comment]
I disagree on the "subjective": there is a `text/markdown` MIME [type](https://tools.ietf.org/html/rfc7763), and using markdown in `text/plain` is surprising. But let's agree to disagree :) I understand this is too much code for just one guy (for now).
